build.sbt

```diff
@@ -64,8 +71,18 @@ val collections =
       ).toList
     )
 
+val collectionsJVM = collections.jvm
+val collectionsJS = collections.js
+
+val root = project.in(file("."))
+  .settings(commonSettings ++ disablePublishing)
```

Comment:
So now the new way to publish is to invoke `; collectionsJVM/publishSigned; collectionsJS/publishSigned`? Would it be possible to make a “short-cut” like you did for `test in Test`?

Reply:
Sure, we can add all the short-cuts you want. (at least for regular tasks; for input tasks it's more complicated)

Reply:
Done.